src/Testcontainers/Clients/DockerContainerOperations.cs

```diff
@@ -62,6 +63,25 @@ public async Task<long> GetExitCode(string id, CancellationToken ct = default)
         .ConfigureAwait(false)).StatusCode;
     }
 
+    public async Task<(string Stdout, string Stderr)> GetLogs(string id, TimeSpan since, TimeSpan until, CancellationToken ct = default)
+    {
+      var logsParameters = new ContainerLogsParameters
+      {
+        ShowStdout = true,
+        ShowStderr = true,
+        Since = since.TotalSeconds.ToString("0", CultureInfo.InvariantCulture),
+        Until = until.TotalSeconds.ToString("0", CultureInfo.InvariantCulture),
+        Timestamps = true,
+      };
+
+      using (var stdOutAndErrStream = await this.Docker.Containers.GetContainerLogsAsync(id, false, logsParameters, ct)
+        .ConfigureAwait(false))
+      {
+        return await stdOutAndErrStream.ReadOutputToEndAsync(ct)
+          .ConfigureAwait(false);
+      }
+    }
```

Comment:
I think it can be simplified
```suggestion
      using var stdOutAndErrStream = await this.Docker.Containers.GetContainerLogsAsync(id, false, logsParameters, ct).ConfigureAwait(false);
      return await stdOutAndErrStream.ReadOutputToEndAsync(ct).ConfigureAwait(false);
    }
```

Reply:
You are right, but I do not like `using`s without brackets. I would like to "decide" when it should be disposed (it is easier to read and helps to avoid mistakes IMO).